files/windows/alias.psm1

```diff
@@ -0,0 +1,21 @@
+## Name: C:\Users\Administrator\Documents\WindowsPowerShell\Modules\alias\alias.psm1
+
+Function validate_erb_func {
+ [CmdletBinding()]
+   Param($arg)
+   $pwd = Get-Location
+   $filename = "$pwd\$arg"

```

Comment:
@Iristyle What kind of issues can you run into with Get-Location? Looks like there are a few ways to do it. Do you typically have to worry about the Powershell version when retrieving the script location? Some of the replies in this thread look similar to your code snippet: http://stackoverflow.com/a/5466355 Would you still use the one you suggested?